### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mherrmann**
The packages in this repository are not easily browseable online. This PR updates the README to add a link to https://winstall.app, a free and and open source web interface for browsing the packages. The PR also changes the position of one paragraph, to make the structure a little clearer.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121072)

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
It looks good

**Comment by stephengillie** [comment]
Hi @mhermmann,

The [THIRD_PARTY.md](https://github.com/microsoft/winget-pkgs/blob/master/THIRD_PARTY.md) file might be a better place to add these details.

**Comment by mherrmann** [comment]
Hi @stephengillie,

thank you for your reply. I believe the way most users come to this repository is when they search for "winget repository". Then they see its rather technical contents and are confused. The change I am proposing fixes this. Similar to your suggestion, it mentions that winstall is a third-party service. But it is much clearer than a nested file. winstall is free and open source. It is updated every 15 minutes with the [latest packages](https://winstall.app/apps) and thus gives a very nice real-time view into the winget repository. Making the change I am proposing would really benefit users. Would you please consider these points?

Thank you!
Michael

File: README.md [hunk 1]
```diff
@@ -5,15 +5,17 @@
 [![Publish Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=PublishPipeline 'Publish Pipeline Badge')](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338)
 [![GitHub Status](https://img.shields.io/endpoint?url=https://api.bittu.eu.org/github-status-badge-endpoint)](https://www.githubstatus.com)
 
-This repository contains the manifest files for the **Windows Package Manager** default source. You are highly encouraged to submit manifests for your favorite application.
-
-> [!IMPORTANT]
-> At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
+This repository contains the manifest files for the **Windows Package Manager** default source. To browse the available packages, you can for example use the third-party service [winstall](http://winstall.app).
```
Comment (stephengillie) [inline README.md:8]:
```suggestion
This repository contains the manifest files for the **Windows Package Manager** default source.
```

Comment (stephengillie) [inline README.md:8]:
```suggestion
This repository contains the manifest files for the **Windows Package Manager** default source. To browse the available packages, you can use the third-party services listed in [THIRD_PARTY.md](https://github.com/microsoft/winget-pkgs/blob/master/THIRD_PARTY.md).
```

**Comment by mherrmann** [comment]
I've updated my PR. What do you think @stephengillie?

**Comment by mherrmann** [comment]
Hi @stephengillie, have you had a chance to look at my changes?

**Comment by stephengillie** [comment]
> Hi @mhermmann,
> 
> The [THIRD_PARTY.md](https://github.com/microsoft/winget-pkgs/blob/master/THIRD_PARTY.md) file might be a better place to add these details.

File: THIRD_PARTY.md [hunk 2]
```diff
@@ -7,3 +7,9 @@ Microsoft is working with multiple partners to develop protocols or an API to en
 - **[WinGetty](https://wingetty.dev)** open source rest server for creating a private repository
```
Comment (stephengillie) [inline THIRD_PARTY.md:8]:
```suggestion
**[winstall.app](https://winstall.app)** is updated every 15 minutes and offers an easy way to search for packages in the winget repository.
```

File: THIRD_PARTY.md [hunk 3]
```diff
@@ -7,3 +7,9 @@ Microsoft is working with multiple partners to develop protocols or an API to en
...
```
Comment (stephengillie) [inline THIRD_PARTY.md:7]:
```suggestion
- **[WinGetty](https://wingetty.dev)** open source rest server for creating a private repository
**[winstall.app](https://winstall.app)** is updated every 15 minutes and offers an easy way to search for packages in the winget repository.
```

File: THIRD_PARTY.md [hunk 4]
```diff
@@ -7,3 +7,9 @@ Microsoft is working with multiple partners to develop protocols or an API to en
...
 
 - More coming soon. Please feel welcome to submit a pull request to add your company's third-party repository offering to this list.
+
+# Web sites for browsing the Community Repository
+
+The following web sites list the packages in the Community Repository:
+
+- **[winstall.app](https://winstall.app)** is updated every 15 minutes and offers an easy way to search for packages in the winget repository.
```
Comment (stephengillie) [inline THIRD_PARTY.md:15]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -5,15 +5,17 @@
...
 The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).
 
 # Submitting a Package
 
+You are highly encouraged to submit manifests for your favorite application.
+
+> [!IMPORTANT]
+> At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
+
 To submit a package to this repository, you should follow these steps:
```
Comment (stephengillie) [inline README.md:19]:
```suggestion
```

File: THIRD_PARTY.md [hunk 6]
```diff
@@ -5,5 +5,6 @@ Microsoft is working with multiple partners to develop protocols or an API to en
 
 - **[winget.pro](https://winget.pro)** offers private winget repository hosting on a paid basis.
 - **[WinGetty](https://wingetty.dev)** open source rest server for creating a private repository
+- **[winstall.app](https://winstall.app)** is updated every 15 minutes and offers an easy way to search for packages in the winget repository.
```
Comment (Trenly) [inline THIRD_PARTY.md:8]: This doesn't appear to be a third party repository (repository in this case is a place where winget packages are hosted), but instead is a third party tool for browsing winget packages. This should be in a separate section and have it's own header - something like `Third Party Tools`

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**